src/callbacks_step/alive.jl

```diff
@@ -86,9 +86,14 @@ function (alive_callback::AliveCallback)(integrator)
         println("─"^100)
         println()
     elseif mpi_isroot()
+        t = integrator.t
+        t_final = last(integrator.sol.prob.tspan)
+        sim_time_percentage = t / t_final * 100
```

Comment:
This assumes that you start at `t = 0`. However, you can have a different starting point. Thus, you should probably check something like

```suggestion
        t = integrator.t
        t_initial = first(integrator.sol.prob.tspan)
        t_final = last(integrator.sol.prob.tspan)
        sim_time_percentage = (t - t_initial) / (t_final - t_initial) * 100
```

Reply:
Done